asv_setup/launch/ASV_simulator.launch.py

```diff
@@ -0,0 +1,35 @@
+import os
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.actions import SetEnvironmentVariable, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+    # LQR_controller launch
+    lqr_controller_launch = IncludeLaunchDescription(
+    PythonLaunchDescriptionSource(
+            os.path.join(get_package_share_directory('lqr_controller'), 'launch/lqr_controller.launch.py')
```

Comment:
Nitpick, but the os.path.join takes care of delimiters etc, so these should ideally be:

`os.path.join(get_package_share_directory('lqr_controller'), 'launch', 'lqr_controller.launch.py')`

etc